Dockerfile

```
@@ -1,10 +1,10 @@
 FROM registry.cloudogu.com/official/base:3.20.2-1 as swaggerui
```

This image could be newer with fewer CVEs.
```suggestion
FROM registry.cloudogu.com/official/base:3.20.3-2 as swaggerui
```